# Generated by Django 3.0.6 on 2020-07-03 05:06

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('exception', '0010_auto_20200703_0504'),
    ]

    operations = [
        migrations.AlterField(
            model_name='validate_exception',
            name='approver',
            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='approverValidate', to=settings.AUTH_USER_MODEL),
        ),
    ]
